specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/serviceTags.json

```diff
@@ -73,6 +73,49 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/locations/{location}/serviceTags/paged": {
```

Comment:
I'm not sure you should added new API for the paged version of serviceTags. Please look at other examples in NRP to evolve the existing API to support paging.

Reply:
So, as I discussed with you during office hours, https://stackoverflow.microsoft.com/questions/225823 , the current ServiceTag definition returned by api is a single object of structure:

public class ServiceTag {

    public string Name { get; set; }

    public string Id { get; set; }

    public string Type { get; set; }

    public string ChangeNumber { get; set; }

    public string Cloud { get; set; }

    public List<ServiceTagInformation> Values { get; set; }
}

And, the requirement is to paginate over the Values list , and not ServiceTag. 
Hence, I plan to add a new definition ServiceTagsDetail, as per the PR.
If I change the definition returned by old api, it would break the existing workflow.
What do you suggest?

Reply:
IMO adding a query parameter to get the optimal response would be better